File: eps-nwt-backend/src/main/java/com/example/epsnwtbackend/dto/AllRealEstateRequestsDTO.java
package com.example.epsnwtbackend.dto;

import com.example.epsnwtbackend.enums.RealEstateRequestStatus;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class AllRealEstateRequestsDTO {
    private Long id;
    private Long owner;
    private RealEstateRequestStatus status;
    private Date createdAt;
    private Date finishedAt;
    private String address;
    private String municipality;
    private String town;
}

File: eps-nwt-frontend/src/app/model/all-real-estate-requests-dto.ts
import {RealEstateRequestStatus} from "../enum/real-estate-request-status";

export interface AllRealEstateRequestsDto{
  id? : number;
  owner : number;
  status? : RealEstateRequestStatus;
  createdAt: Date | null;
  finishedAt: Date | null;
  address: string;
  municipality: string;
  town: string;
}
